python/graphstorm/model/utils.py

```diff
@@ -738,7 +756,108 @@ def save_shuffled_node_embeddings(shuffled_embs, save_embed_path, save_embed_for
 
     if rank == 0:
         with open(os.path.join(save_embed_path, "emb_info.json"), 'w', encoding='utf-8') as f:
-            f.write(json.dumps(emb_info))
+            json.dump(emb_info, f, indent=4)
+
+def save_full_node_embeddings(g, save_embed_path,
+                              embeddings,
+                              node_id_mapping_file,
+                              save_embed_format="pytorch"):
+    """ Save the entire node embeddings as pytorch tensors in
+        a distributed way.
+
+        The input node `embeddings` are stored in Partition Node ID space.
+        By default, `save_full_node_embeddings` will translate the node IDs
+        from Partition Node ID space into their counterparts in Graph Node
```

Comment:
i thought it will store node embeddings with the raw node IDs?

Reply:
In Graph ID space.